src/solvers/LevenbergMarquardt.jl

```diff
@@ -167,6 +167,8 @@ function LevenbergMarquardt!(
                                           StopWhenStepsizeLess(1e-12),
     debug=[DebugWarnIfCostIncreases()],
     expect_zero_residual::Bool=false,
+    initial_residual_values=similar(p, get_objective(nlso).num_components),
+    initial_jacF=similar(p, get_objective(nlso).num_components, manifold_dimension(M)),
```

Comment:
```suggestion
    initial_jacobian_F=similar(p, get_objective(nlso).num_components, manifold_dimension(M)),
```

internally shortening it might be ok, but I feel interfaces (so also keyword arguments) should be as descriptive as possible and only in rare cases abbreviate (remember to update the other code line as well)